simapp/sim_test.go

```diff
@@ -56,17 +56,6 @@ func testAndRunTxs(app *SimApp, config simulation.Config) []simulation.WeightedO
 
 	// nolint: govet
 	return []simulation.WeightedOperation{
-		{
-			func(_ *rand.Rand) int {
-				var v int
-				ap.GetOrGenerate(app.cdc, OpWeightDeductFee, &v, nil,
-					func(_ *rand.Rand) {
-						v = 5
-					})
-				return v
-			}(nil),
-			authsimops.SimulateDeductFee(app.AccountKeeper, app.SupplyKeeper),
```

Comment:
no need for this anymore as we use the standard tx cycle through the anteHandler